frontend/src/components/widget/DatetimeRange.js

```diff
@@ -205,6 +225,11 @@ DatetimeRange.propTypes = {
   mandatory: PropTypes.any,
   validStatus: PropTypes.any,
   timePicker: PropTypes.any,
+  isFilterActive: PropTypes.bool,
+  defaultValue: PropTypes.string,
```

Comment:
You can remove `value/valueTo` from props

Reply:
not quite we use defaultValue, defaultVauleTo as default ones and value,/valueTo as the normal ones